tests/forms/gridfield/GridFieldFilterHeaderTest.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+class GridFieldFilterHeaderTest extends SapphireTest {
+
+	protected $extraDataObjects = array(
+		'GridFieldFilterHeaderTest_DataObject',
+	);
+
+	public function testColumnToFilterField() {
+		$class = 'GridFieldFilterHeaderTest_DataObject';
+		$header = new GridFieldFilterHeader();
+		$method = new ReflectionMethod($header, 'columnToFilterField');
+		$method->setAccessible(true);
+		$this->assertEquals('Title', $method->invoke($header, $class,'Title.ATT'));
```

Comment:
I think this will be fine, actually, if the first part is a relation it just returns the whole thing (current behaviour).